test_cases/state_totals/out/Contract.approval.teal

```diff
@@ -7,17 +7,8 @@ test_cases.state_totals.contract.Contract.approval_program:
     //     state_totals=StateTotals(local_bytes=1, global_uints=GLOBAL_UINTS),
     // ):
     txn NumAppArgs
-    bz main_bare_routing@4
-    txna ApplicationArgs 0
-    match
```

Comment:
This appears to be the only change to an optimised teal file, and it actually fixes a bug we'd missed where an empty `match` is created

Reply:
Haha, looks like it still runs, but yeah it's effectively a no-op that costs 2 ops 😬